README.md

```diff
@@ -41,6 +41,22 @@ Full installation instructions are available in [INSTALL.md](https://github.com/
 
 ## Working with data
 
+### Importing
+
+AcousticBrainz provides data dumps that you can import into your own server.
+Latest database dump is available at http://acousticbrainz.org/download. You
+need to download full database dump from this page and use it during database
+initialization:
+
+    $ python manage.py init_db -a <path_to_the_archive>
+
+or import archive after database is created:
+
+    $ python manage.py import_data -a <path_to_the_archive>
```

Comment:
After discussing this in IRC, I realised I confused the situations of no database, database with no tables, and database with tables. I propose these commands:

 - createdb and tables to run your own server
 - createdb, make tables, import when no db exists previously
 - if the database exists, add a force or drop option to delete the database first before doing the above
 - import a dump into already created database and tables (for updates, not recommended for initial import because of the indexes)

Only the third option is missing, I believe.